rpcs3/Crypto/aes.cpp

```diff
@@ -31,6 +31,20 @@
 
 #include "aes.h"
 
+#include <bitset>
+#include <cstdio>
+
+// Returns whether AES-NI is supported
+static bool AESNI() {
+	int cpuInfo[4];
+#ifdef _WIN32

```

Comment:
@cornytrace There are a lot of compilers on Windows, other than MSVC (MinGW gcc/clang,Cygwin gcc/clang, Clang/C2 etc.)